nbgrader/html/templates/formgrade.tpl

```diff
@@ -64,6 +65,9 @@ window.MathJax = {
 {%- endblock body %}
 
 {% block footer %}
+<script>
+    autosize(document.querySelectorAll('textarea'));
```

Comment:
I think a better place to actually do the autosize would be in the `initialize` function for the `CommentUI` (which is in `models.js`).